### PR TITLE
Add 3.6 bundles

### PR DESCRIPTION
**Pull request by brianhlin**

### CODE REVIEW

File: bundles.ini [hunk 1]
```diff
@@ -123,6 +136,20 @@ packages    = osg-wn-client osg-update-data osg-ca-scripts
...
+[osg-afs-client-3.6]
+paramsets   = el7,x86_64
+patchdirs   = patches/wn-client/common
+              patches/wn-client/common/%(dver)s
+              patches/wn-client/3.6/%(dver)s
+              patches/afs-client/3.6/common
+              patches/afs-client/3.6/%(dver)s
+dirname     = osg-afs-client
+tarballname = osg-afs-client-%(version)s-%(relnum)s.%(dver)s.%(basearch)s.tar.gz
+packages    = osg-wn-client osg-update-data osg-ca-scripts
```
Comment (matyasselmeci) [inline bundles.ini:148]: I think we don't have osg-update-data.

**Review by timtheisen — APPROVED** [review]
LGTM

File: bundles.ini (hunk 1 above)
Comment (matyasselmeci) [reply bundles.ini:148]: Never mind, it's part of osg-update-vos.

**Review by matyasselmeci — APPROVED** [review]

File: bundles.ini (hunk 1 above)
Comment (brianhlin) [reply bundles.ini:148]: We should, does it have to be in release or something? https://koji.chtc.wisc.edu/koji/buildinfo?buildID=14298

**Pull request merged**